src/Api/ProductMediaFileApi.php

```diff
@@ -132,12 +132,20 @@ protected function extractCodeFromCreationResponse(ResponseInterface $response)
     {
         $headers = $response->getHeaders();
 
-        if (!isset($headers['Location'][0])) {
+        if (isset($headers['Location'][0])) {
```

Comment:
there's a nicer way to do this IMO, by using the [hasHeader](https://github.com/php-fig/http-message/blob/efd67d1dc14a7ef4fc4e518e7dee91c271d524e4/src/MessageInterface.php#L78) and [getHeader](https://github.com/php-fig/http-message/blob/efd67d1dc14a7ef4fc4e518e7dee91c271d524e4/src/MessageInterface.php#L94) methods, which are case insensitive
Smth like:
```php
if (!$response->hasHeader('location')) {
    throw new RuntimeException('...');
}

$locationHeader = $response->getHeader('location')[0];
```

Can you update the PR accordingly please? (you'll also need to update the [associated test](https://github.com/akeneo/api-php-client/blob/master/spec/Api/ProductMediaFileApiSpec.php) :slightly_smiling_face: )